app/adapters/rating.js

```diff
@@ -0,0 +1,6 @@
+import DS from 'ember-data';
+
+export default DS.RESTAdapter.extend({
+  // host: 'http://skill-inventory.herokuapp.com',
```

Comment:
Yea, I have that, but setting host: 'http://skill-inventory.herokuapp.com' or host = 'http://skill-inventory.herokuapp.com' doesn't work